sg_covid_impact/utils/altair_save_utils.py

```diff
@@ -0,0 +1,46 @@
+from altair_saver import save
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+import os
+
+import sg_covid_impact
+
+project_dir = sg_covid_impact.project_dir
+
+FIG_PATH = f"{project_dir}/figures"
+
+# Checks if the right paths exist and if not creates them when imported
+if not os.path.exists(f"{FIG_PATH}/png"):
+    os.mkdir(f"{FIG_PATH}/png")
+
+if not os.path.exists(f"{FIG_PATH}/html"):
+    os.mkdir(f"{FIG_PATH}/html")
+
+
+def altair_visualisation_setup():
+    # Set up the driver to save figures as png
+    driver = webdriver.Chrome(ChromeDriverManager().install())
```

Comment:
This doesn't work on my system because I have Chromium rather than Chrome, no biggie because I can initalise and pass in my own driver; however I'd suggest naming this to `google_chrome_driver_setup` to reflect the fact that it doesn't do anything with altair